### ISSUE TITLE
How to check accessibility disabled controls from the project?

### ISSUE THREAD

**Issue by KarthikaThevan** [realm/SwiftLint#3479]
By using swiftlint how to check the views and controls are accessibility enabled or disabled while building the project. 
We need a rule for finding the accessibility disabled UIcontrols in a build time. Kindly help us to achieve this feature.

**Comment by KarthikaThevan** [comment]
Hi Team,

We are having ebook reading application developed in IOS platform. We want to use swiftLint tool to check whether accessibility enabled for all UIControls which are in storyboard as well as programmatically initialised controls.

Our requirement: We are supporting voice over for visually disabled people. Accessibility should be enabled for all UI controls (eg : UIButton, UILabel… etc) in application to support voice over.

Use case : When we make a build, if the controls doesn’t have accessibility enabled true property, then the build should fail in Xcode. SwiftLint tool should fail builds if accessibility tags are not present for UIControls in the application.

Kindly guide us how to get all UIElements in the application and how to check the property values of the UIElement instance.
For example, get all the UILabels and check whether the UILabels are set as IsAccessibilityElement is True. Likewise, we need to get all UIElements and check the IsAccessibilityElement property value for each UIElement instance.

Could you please suggest on the above so that, we can try our best to implement it in SwiftLint.

**Comment by jpsim** [comment]
See #3508 and #3484
